app/helpers/remote_execution_helper.rb

```diff
@@ -236,4 +243,25 @@ def load_template_from_task(template_invocation, target)
     return if task.nil?
     task.execution_plan.actions[1].try(:input).try(:[], 'script')
   end
+
+  def jobs_template_with_input(job_invocation)
+    template_name = Setting['remote_execution_job_invocation_template']
```

Comment:
I'd combine this and the next line, but not a strong opinion, IMHO one extra variable is overkill for this, it's not used anywhere else